examples/rust/poseidon/src/main.rs

```diff
@@ -0,0 +1,54 @@
+use icicle_bls12_381::poseidon;
```

Comment:
Getting an unused import warning from this

Reply:
Removed